ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/rlpx/connections/netty/DeFramer.java

```diff
@@ -109,12 +109,12 @@ protected void decode(final ChannelHandlerContext ctx, final ByteBuf in, final L
         try {
           peerInfo = HelloMessage.readFrom(message).getPeerInfo();
         } catch (final RLPException e) {
-          LOG.debug("Received invalid HELLO message", e);
+          LOG.warn("Received invalid HELLO message, set log level to TRACE for message body", e);
```

Comment:
Are we sure that passing this log in warning does not risk spamming the logs of a node running on the mainnet or on ropsten etc ? Do we often receive invalid messages of this type ?

Reply:
Great point. I looked at log aggregation for all of our canaries and only see 1 occurrence of this exception. This should not result in logspam at warn instead of debug.

Reply:
ok 👍